.github/workflows/release-new-charts.yaml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - master
+      - dm-tweaks-for-clinvar-DSPDC-1176
```

Comment:
Testing out what I hope will be an easy trick for publishing WIP charts to test with